src/main/java/bio/terra/common/exception/SamApiException.java

```diff
@@ -22,6 +22,10 @@
   private static Logger logger = LoggerFactory.getLogger(SamApiException.class);
 
   public static SamApiException create(ApiException apiException) {
+    return SamApiException.create("", apiException);
```

Comment:
nit: avoid using "magic values" like the empty string. Either use null or Optional empty if there's no valid messagePrefix.

Reply:
done